snapshots/output/basic-project/test/basic_test.dart

```diff
@@ -0,0 +1,15 @@
+  import 'package:test/test.dart';
+// definition scip-dart pub dart_test 1.0.0 test/basic_test.dart/
+  
+  void main() {
+//     ^^^^ definition scip-dart pub dart_test 1.0.0 test/basic_test.dart/main().
+//     documentation ```dart
+    group('some test', () {
+//  ^^^^^ reference scip-dart pub test_core 0.5.3 lib/scaffolding.dart/group().
+      test('equality', () {
+//    ^^^^ reference scip-dart pub test_core 0.5.3 lib/scaffolding.dart/test().
+        expect(1, 1);
+//      ^^^^^^ reference scip-dart pub matcher 0.12.16 lib/src/expect/expect.dart/expect().
```

Comment:
Since you are no longer checking in the pubspec.lock file, will these package versions cause test failures when new versions of packages like test_core and matcher are released?

Reply:
Thats a fantastic point, there wasnt really a good reason for removing that, i'll re-add to keep the CI consistent